students/itsabhi/Lesson4/codekata.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python3
+
+
+def wordmap(fh):
+    # This function creates a dictionary of trigram sequence
+    oneline = ""
+    while True:
+        line = fh.readline()
+        oneline += line.rstrip('\r')
+        if not line:
+            break
+
+    print(oneline)
+    word_list = oneline.split()
+    n = 0
+    word_dict = {}
+    while(n+2 < len(word_list)):
+        f_word = word_list[n]
+        s_word = word_list[n+1]
+        t_word = word_list[n+2]
+
+        if (f_word + " " + s_word) not in word_dict:  # If key exists in dict
```

Comment:
This seems to be the wrong comment here, I'm guessing you rearranged the order of the conditional statements at some point? It's a good idea in general to use comments sparingly, aiming instead to write code that explains itself, because it's really easy to leave misleading comments like this lying around